addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -361,6 +361,22 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
         --o-wsale-card-padding: #{$o-wsale-products-layout-grid-gutter-width * .5};
         --o-wsale-card-info-padding: #{map-get($spacers, 2)} #{map-get($spacers, 3)};
     }
+
+    .oe_advanced_configurator_modal {
+        tr {
+            display: flex;
+            flex-wrap: nowrap;
+            justify-content: space-around;
+        }
+
+        td, th {
+            flex: 1;
+        }
+
+        thead {
+            display: block;
+        }
```

Comment:
I don't understand these values, as far as I can tell the issue is caused by this CSS:

https://github.com/odoo/odoo/blob/0bd8146dbe4f884d2744e6ce38f2341d8e7cded3/addons/website_sale/static/src/scss/website_sale.scss#L7-L25

that target erroneously the table inside the modal, if it's the case it would make sense to just counter the CSS.  For example:

```suggestion
        table, tbody, tr, td {       
            display: revert;  
        }
```
or this could be done in the mixin directly:

```diff
--- a/addons/website_sale/static/src/scss/website_sale.scss
+++ b/addons/website_sale/static/src/scss/website_sale.scss
@@ -21,6 +21,12 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
             display: if($-list, block, inline-block);
             width: if($-list, 100%, 50%);
         }
+
+        .modal-dialog {
+            table, tbody, tr, td {
+                display: revert;
+            }
+        }
     }
 }
```
or another solution is to have the mixin be more stringent and forcing that the table is a direct child:

```diff
--- a/addons/website_sale/static/src/scss/website_sale.scss
+++ b/addons/website_sale/static/src/scss/website_sale.scss
@@ -6,20 +6,21 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
 
 @mixin wsale-break-table($-list: false) {
     .o_wsale_products_grid_table_wrapper {
-        table, tbody {
-            display: block;
-            width: 100%;
-        }
-
-        tr {
-            display: if($-list, block, flex);
-            width: 100%;
-            flex-wrap: wrap;
-        }
-
-        td {
-            display: if($-list, block, inline-block);
-            width: if($-list, 100%, 50%);
+        > table {
+            &, > tbody {
+                display: block;
+                width: 100%;
+
+                > tr {
+                    display: if($-list, block, flex);
+                    width: 100%;
+                    flex-wrap: wrap;
+                    > td {
+                        display: if($-list, block, inline-block);
+                        width: if($-list, 100%, 50%);
+                    }
+                }
+            }
         }
     }
 }
```
The last two solutions seems the best, but if done, you should also fix the modal on mobile, because by a side effect the mobile view in current version looked OK when grid view was set (because the style of the mobile grid view was set on the modal), so there should be something like (`.modal-body { ... override the style of the table so it looks good on mobile }`)